prow/plank/controller.go

```diff
@@ -598,4 +606,49 @@ func (c *Controller) patchProwjob(srcPJ prowapi.ProwJob, destPJ prowapi.ProwJob)
 	_, err = c.prowJobClient.Patch(srcPJ.Name, ktypes.MergePatchType, patch)
 	c.log.WithFields(pjutil.ProwJobFields(&destPJ)).Debug("Patched ProwJob.")
 	return err
+
+}
+
+// Get the jobs that we need to retain so horologium can continue working
+// as intended.
+// And clean up old pods.
+func (c *Controller) cleanUpPods() {
```

Comment:
Please also move the metrics (podsCreated, podsRemoved, podsRemovalErrors) over instead of just removing them